src/Symfony/Component/Workflow/Validator/WorkflowValidator.php

```diff
@@ -20,5 +20,6 @@ class WorkflowValidator implements DefinitionValidatorInterface
 {
     public function validate(Definition $definition, $name)
     {
+        return true;
```

Comment:
Ok I just remove this extra commit to keep the focus on the config fix in this PR.